packages/pymedphys_pinnacle/src/pymedphys_pinnacle/exporttool/pinnacle_image.py

```diff
@@ -1,3 +1,53 @@
+# Copyright (C) 2019 South Western Sydney Local Health District,
+# University of New South Wales
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+# This work is derived from:
+# https://github.com/AndrewWAlexander/Pinnacle-tar-DICOM
```

Comment:
Might even be helpful to include the commit hash in this link (and if you're keen the file path this particular code was derived from).

It's certainly not necessary, but it enables someone to go and use/compare to the original code if they wish.

Certainly not a must. Up to you.

Reply:
For example https://github.com/pymedphys/pymedphys/blob/master/app/src/components/select-script.tsx#L27-L28